### ISSUE TITLE
Keyboard popping up (ionic 3.2)

### ISSUE THREAD

**Issue by xthenth** [jasonmamy/cordova-wheel-selector-plugin#29]
Is it possible to disable keyboard popping up upon tapping on element?

For example I chose an item on the wheel picker then i tapped on it (or any item on the wheel picker), there comes the keyboard popping up and allows me to edit the value of the item

ionic version
3.20.0

**Comment by jasonmamy** [comment]
Yes, it looks like on Android I can disable the editing of the fields using:

picker.setDescendantFocusability(NumberPicker.FOCUS_BLOCK_DESCENDANTS);

It didn't seem to stop the keyboard from popping up, I didn't see any code
to do that.

I re-released as 1.1.1 (just re-install from:
https://ionicframework.com/docs/native/wheelselector-plugin/).
